hystrix-contrib/hystrix-codahale-metrics-publisher/src/test/java/com/netflix/hystrix/contrib/codahalemetricspublisher/HystrixCodaHaleMetricsPublisherCommandTest.java

```diff
@@ -20,27 +23,28 @@ public void setup() {
         HystrixPlugins.getInstance().registerMetricsPublisher(new HystrixCodaHaleMetricsPublisher(metricRegistry));
     }
 
-    @After
-    public void teardown() {
-        HystrixPlugins.reset();
+    @Test
+    public void testCommandSuccess() throws InterruptedException {
+        Command command = new Command();
+        command.execute();
+
+        Thread.sleep(1000);
```

Comment:
surely you can use await here instead of sleep from https://github.com/awaitility/awaitility
such as   
```
Gauge<Long> successGauge = metricRegistry.getGauges().get("test.test.countSuccess");
await().atMost(1, SECONDS).until(successGauge::getValue, isEqualTo(1L));
```